distributions/openhab/src/main/resources/userdata/etc/org.ops4j.pax.web.cfg

```diff
@@ -6,7 +6,7 @@ org.ops4j.pax.web.server.minThreads = 2
 #
 # max server threads
 #
-org.ops4j.pax.web.server.maxThreads = 50
+org.ops4j.pax.web.server.maxThreads = 20
```

Comment:
Why downgrade this?

https://github.com/openhab/openhab-distro/issues/192

Reply:
Good catch - this was actually a left over from some threading analysis, which wasn't meant to be committed. I have reverted it again now!